README.md

```diff
@@ -54,7 +54,7 @@ for items in items_of(client):
   emit("did something")
   # do something else
   emit("did something else")
-  context().stop()
+  context().stop() # or could be context().fail()
```

Comment:
Unless you really want it to be GumShoe.  I don't want to stifle your self-expression :)